### DISCUSSION TITLE
AVIF image support

### DISCUSSION THREAD

**Discussion by AlphaJack** [spacecowboy/Feeder#269] [Ideas]
Hi, can we have avif support?
It is a new image format now supported by all major browsers: https://caniuse.com/avif

**Comment by spacecowboy** [comment]
Then please redirect your feature request to https://coil-kt.github.io/coil/

**Reply by AlphaJack** [reply]
The developer says that it should be supported from Android 12+: https://github.com/coil-kt/coil/issues/1666#issuecomment-1472944138

**Reply by spacecowboy** [reply]
well there you are then.
